packages/dds/tree/src/schema-stored/storedSchemaRepository.ts

```diff
@@ -3,62 +3,47 @@
+     */
+    readonly policy: TPolicy;
+}
+
 /**
- * Example in memory SchemaRepository showing how stored schema could work.
- *
- * Actual version for use with Fluid would probably need to either be copy on write, support clone,
- * have rollback/rebase of updates to handle local changes before they are sequenced.
- *
- * New instances default to permitting only empty documents.
- * Assuming a conventional `FieldSchemaIdentifier` use for the document root,
- * this default state can be incrementally updated permitting more and more possible documents.
- *
- * All states which were ever permitted (including the empty document) will remain valid for all time.
- * This means that updating the schema can be done without access to the document contents.
- * As long as the document content edits keep it in schema with the current (or any past) schema,
- * it will always be in schema for all future schema this StoredSchemaRepository might contain.
- *
- * This approach means that stored schema can be updated
- * to permit data in new formats without having to look at any document content.
- * This is valuable for large document scenarios where no user has the entire document loaded,
- * but still need to add some new types to the document.
- *
- * The ergonomics issues caused by the stored schema permitting all old
- * date-formats can be addressed be using a schema on read system (schematize)
- * to apply a more restricted "view schema" when reading document content.
- *
- * The above design pattern (combining stored and view schema this way to support
- * partial checkouts with stored schema that can be updated)
- * is the intended usage pattern for typical users, but other configurations of these systems are possible:
- * Systems which have access to the document content could permit additional kinds of schema changes.
- * For example, a system which keeps the whole document in memory,
- * or is willing to page through all the data when doing the change could permit:
- * - arbitrary schema changes as long as all the data currently complies
- * - schema changes coupled with instructions for how to updated old data
- * While this is possible,
- * it is not the focus of this design since such users have strictly less implementation constraints.
+ * Mutable collection of stored schema.
  *
  * TODO: could implement more fine grained dependency tracking.
- *
- * @sealed
  */
-export class StoredSchemaRepository<TPolicy extends SchemaPolicy = SchemaPolicy>
-    extends SimpleDependee implements SchemaData {
+export interface StoredSchemaRepository<TPolicy extends SchemaPolicy = SchemaPolicy>
+    extends Dependee, SchemaDataAndPolicy<TPolicy> {
+    /**
+     * Add the provided schema, possibly over-writing preexisting schema.
+     */
+    update(newSchema: SchemaData): void;
+}
+
+/**
+ * StoredSchemaRepository for in memory use:
+ * not hooked up to Fluid: does not create an FLuid Ops when editing.
```

Comment:
```suggestion
 * Not hooked up to Fluid and does not create any FLuid ops when editing.
```

Reply:
grammatically, you are not supposed to capitalize a clarifying phrase introduced by a ":".

Also the second colon is introducing a clarifying phrase to the clarifying phrase from the first one, so that phrase should not be at the same level.

That said its pretty unclear, so I have improved it.